project/src/demo/ui/chat/MoodDemo.tscn

```diff
@@ -51,9 +54,11 @@ dna = {
 "line_rgb": "41281e",
 "mouth": "1",
 "nose": "3",
-"property:Body/Viewport/Body/NeckBlend:frame_data": "res://assets/main/world/creature/1/neck-packed.json",
+"property:Body/Viewport/Body/NeckBlend:frame_data": "res://assets/main/world/creature/1/neck-blend-packed.json",
```

Comment:
Document this renaming in the commit comment.

Reply:
Added comment, `avoid ambiguity with collar resources, as the collar is also neck-related`